scripts/packages/deptry_reproducer/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "deptry_reproducer"
```

Comment:
Can we put this in `editable-installs`, just because that's where all our other local packages are?

Reply:
Do you mind if i rename editable installs to `packages` first in a separate PR?

Reply:
Yeah, that’s fine.